membrane_curvature/tests/test_mdakit_membcurv.py

```diff
@@ -163,24 +163,121 @@
+               13092, 13104, 13116, 13128, 13140, 13152, 13164, 13176, 13188, 13200,
+               13212, 13224, 13236, 13248, 13260, 13272, 13284, 13296, 13308, 13320,
+               13332, 13344, 13356, 13368, 13380, 13392, 13404, 13416, 13428, 13440,
+               13452, 13464, 13476, 13488, 13500, 13512, 13524, 13536, 13548, 13560,
+               13572, 13584, 13596, 13608, 13620, 13632, 13644, 13656, 13668, 13680,
+               13692, 13704, 13716, 13728, 13740, 13752, 13764, 13776, 13788, 13800,
+               13812, 13824, 13836, 13848, 13860, 13872, 13884, 13896, 13908, 13920,
+               13932, 13944, 13956, 13968, 13980, 13992, 14004, 14016, 14028, 14040,
+               14052, 14064, 14076, 14088, 14100, 14112, 14124, 14136, 14148, 14160,
+               14172, 14184, 14196, 14208, 14220, 14232, 14244, 14256, 14268, 14280],
+
+      'POPE': [14292, 14304, 14316, 14328, 14340, 14352, 14364, 14376, 14388, 14400, 14412,
+               14424, 14436, 14448, 14460, 14472, 14484, 14496, 14508, 14520, 14532, 14544,
+               14556, 14568, 14580, 14592, 14604, 14616, 14628, 14640, 14652, 14664, 14676,
+               14688, 14700, 14712, 14724, 14736, 14748, 14760, 14772, 14784, 14796, 14808,
+               14820, 14832, 14844, 14856, 14868, 14880, 14892, 14904, 14916, 14928, 14940,
+               14952, 14964, 14976, 14988, 15000, 15012, 15024, 15036, 15048, 15060, 15072,
+               15084, 15096, 15108, 15120, 15132, 15144, 15156, 15168, 15180, 15192, 15204,
+               15216, 15228, 15240, 15252, 15264, 15276, 15288, 15300, 15312, 15324, 15336,
+               15348, 15360, 15372, 15384, 15396, 15408, 15420, 15432, 15444, 15456, 15468,
+               15480, 15492, 15504, 15516, 15528, 15540, 15552, 15564, 15576, 15588, 15600,
+               15612, 15624, 15636, 15648, 15660, 15672, 15684, 15696, 15708, 15720, 15732,
+               15744, 15756, 15768, 15780, 15792, 15804, 15816, 15828, 15840, 15852, 15864,
+               15876, 15888, 15900, 15912, 15924, 15936, 15948, 15960, 15972, 15984, 15996,
+               16008, 16020, 16032, 16044, 16056, 16068, 16080, 16092, 16104, 16116, 16128,
+               16140, 16152, 16164, 16176, 16188, 16200, 16212, 16224, 16236, 16248, 16260,
+               16272, 16284, 16296, 16308, 16320, 16332, 16344, 16356, 16368, 16380, 16392,
+               16404, 16416, 16428, 16440, 16452, 16464, 16476, 16488, 16500, 16512, 16524,
+               16536, 16548, 16560, 16572, 16584, 16596, 16608, 16620, 16632, 16644, 16656,
+               16668, 16680, 16692, 16704, 16716, 16728, 16740, 16752, 16764, 16776, 16788,
+               16800, 16812, 16824, 16836, 16848, 16860, 16872, 16884, 16896, 16908, 16920,
+               16932, 16944, 16956, 16968, 16980, 16992, 17004, 17016, 17028, 17040]}},
+
+    'small': {'lower': {'POPC': [5, 6, 7], 'POPE': [8, 9]}, 'upper': {'POPC': [0, 1, 4], 'POPE': [2, 3]}},
```

Comment:
The upper POPC molecules of [0, 1, 4] have the coordinates:
>  1923POPC   PO4    1   1.361   0.630  16.101
>  2054POPC   PO4    2   0.704   2.229  15.981
>  2622POPC   PO4    5   1.451   0.382  *11.752*

That last POPC looks much more similar to the lower POPC with the coordinates:
>  2702POPC   PO4    6   0.081   0.610  11.835
>  2708POPC   PO4    7   0.678   2.233  11.825
>  2713POPC   PO4    8   0.988   0.554  11.838

Are you sure this is the right answer?